addons/point_of_sale/static/src/js/models.js

```diff
@@ -1539,7 +1544,7 @@ export class Product extends PosModel {
                     return pricelist.id === rule.base_pricelist_id[0];
                 });
                 if (base_pricelist) {
-                    price = self.get_price(base_pricelist, quantity);
+                    price = self.get_price(base_pricelist, quantity, true);
```

Comment:
The method has now 4 parameters `get_price(pricelist, quantity, price_extra, recurring=false)` and I'm pretty sure you are not trying to assign `true` to `extra_price`
```suggestion
                    price = self.get_price(base_pricelist, quantity, undefined, true);
```

Reply:
oO Indeed, thanks for your catch =]